docs/source/data/resolvers.mdx

```diff
@@ -380,8 +379,7 @@ Resolver functions are passed four arguments: `parent`, `args`, `context`, and `
 
 ### The `context` argument
```

Comment:
Should this become `contextValue` or are we broadly keeping things as `context` in the docs?

Reply:
I'll follow up with another subsequent PR for changing this to `contextValue` later. (This is linked in a _ton_ of places and I don't want to flood this PR with little link changes 😂 )